packages/client/src/components/atoms/FormSection/FormSection.tsx

```diff
@@ -28,16 +28,17 @@ const FormSection: React.FC<Props> = ({ name, content }) => {
 
   return (
     <div>
-      {name && <h5 className={classes.sectionTitle}>{name}</h5>}
+      {name && (
+        <h5 className={classes.sectionTitle}>{t(OrganizationLabel[name])}</h5>
+      )}
```

Comment:
Here you're losing the whole point of enums (autocomplete and restricting your choice to enums which actually exist). The name should just be a string and you should take care of the translation when before passing that string. When rendering the component and passing a `name` prop into it.

Reply:
Organization label, down in the code is a different matter and, unfortunately, there's no clever way about it (also, it's an enum to follow the consistency of using enum translations). 
The only way to make this type safe would be to update the `FieldProps` as such:
```typescript
interface FieldProps {
  name: keyof OrganizationData;
  label: keyof typeof OrganizationLabel;
  multiline?: boolean;
}
```

Reply:
So...this way, you wouldn't use any transformations on the name -> label, but would rather read both the name and the label from `FieldProps` and pass the to translation enum, well assured that the enum key, in fact, exists.

Reply:
If this is unclear (as I'm really tired), hit me up tomorrow for further explanation :joy: